Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,439 @@
+{
+    "appid": "00a4cb00-19a1-46f6-9923-f502a3dd346b",
+    "name": "GitLab",
+    "description": "This app integrates with a GitLab instance to perform various lookups and execute CI/CD pipelines.",
+    "type": "devops",
+    "product_vendor": "GitLab",
+    "logo": "gitlab.png",
+    "logo_dark": "gitlab_dark.png",
+    "product_name": "GitLab",
+    "product_version_regex": ".*",
+    "publisher": "Peter Bertel",
+    "license": "Copyright (c) Peter Bertel, 2019",
+    "app_version": "1.0.0",
+    "utctime_updated": "2019-08-26T01:28:27.791651Z",
+    "package_name": "phantom_gitlab",
+    "main_module": "gitlab_connector.py",
+    "min_phantom_version": "4.5.15922",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "gitlab_location": {
+            "description": "The Hostname or IP address of the GitLab instance",
+            "data_type": "string",
+            "required": true,
+            "value_list": [],
```

Comment:
You can delete empty `value_list` and `default`

Reply:
I deleted `value_list` and `default` in [this commit](https://github.com/peterbertel/phantom-apps/commit/8159a235f9d415edc12c6442f09df319460868e2).